utils/src/lib.rs

```diff
@@ -35,30 +35,7 @@ where
     let slot = timestamp / pallet_aura::Pallet::<T>::slot_duration();
     let digest = Digest { logs: vec![DigestItem::PreRuntime(AURA_ENGINE_ID, slot.encode())] };
     frame_system::Pallet::<T>::initialize(&block_number, &Default::default(), &digest);
+    // NOTE: pallet aura only needed with the `parachain` feature
```

Comment:
I thought pallet aura is for standalone mode? 

> pallet aura only needed with the `parachain` feature

This indicates, that pallet aura runs with the parachain feature, which is not true, right?

Reply:
Thanks for correcting!